mla/datasets/base.py

```diff
@@ -51,7 +51,7 @@ def load(dataset="training", digits=np.arange(10)):
 
 
 def load_nietzsche():
-    text = open(get_filename("data/nietzsche.txt")).read().lower() # str Object has already been decoded
+    text = open(get_filename("data/nietzsche.txt", "rt")).read().decode("utf-8").lower()
```

Comment:
You don't need to use decode when `t` is specified.
Please move "rt" to the open function. Currently, you are passing it to the `get_filename`

Reply:
Yeah, you are right! Thanks for the remind. My bad...